kernel/src/syscall/symlink.rs

```diff
@@ -44,6 +45,7 @@ pub fn sys_symlinkat(
 
     let new_path = dir_path.new_fs_child(&link_name, InodeType::SymLink, mkmod!(a+rwx))?;
     new_path.inode().write_link(&target)?;
+    fsnotify_create(&dir_path, link_name)?;
```

Comment:
Rename all Linux-style `fsnotify_xxx` functions to the Rust-style `fs::notify::on_xxx` names to achieve both brevity and clarity. On the user side, the recommended practice is to use the functions without importing functions themselves. For example, this file can be refactored as follows:

```rust
fs::notify::on_create(&dir_path, link_name);
```

Notice how the caller uses the module path rather than the function name to make the code crystal clear.